packages/frontend-2/server/api/webflow.ts

```diff
@@ -0,0 +1,89 @@
+import { ensureError } from '@speckle/shared'
+import type { WebflowItem } from '~/lib/dashboard/helpers/types'
+
+type WebflowApiResponse = {
+  items: Array<{
+    id: string
+    lastPublished: string
+    createdOn: string
+    fieldData: {
+      name: string
+      slug: string
+      'feature-image'?: {
+        url: string
+      }
+      html?: string
+    }
+  }>
+}
+
+const calculateReadTime = (content: string): number => {
+  const wordsPerMinute = 280
+  const wordCount = content.trim().split(/\s+/).length
+  return Math.ceil(wordCount / wordsPerMinute)
+}
+
+// Used to filter to last 6 months' articles to prevent old,
+// recently edited posts from appearing at the top
+const getSixMonthsAgo = (): Date => {
+  const date = new Date()
+  date.setMonth(date.getMonth() - 6)
+  return date
+}
+
+export default defineEventHandler(async (): Promise<{ items: WebflowItem[] }> => {
+  const { webflowApiToken } = useRuntimeConfig()
```

Comment:
What happens if there's no token set? Do we handle that gracefully?

Reply:
I adde an error throw if token missing which should prevent invalid requests. Is this enough here do you think?

Reply:
@andrewwallacespeckle Graceful handling would involve ensuring the UI doesn't break - that outside of those article cards everything else looks normal. I don't think we necessarily need to throw/log any errors, altho we can for informational purposes if it doesn't break the app

Reply:
It doesn't break the app. The whole Blog section will be hidden if there is an error (including no api key)
https://github.com/specklesystems/speckle-server/pull/3342